meerkat_abacus/data_management.py

```diff
@@ -580,6 +580,18 @@ def set_up_database(leave_if_data, drop_db, param_config=config):
         logging.info("Populating DB")
         model.form_tables(param_config=param_config)
         model.Base.metadata.create_all(engine)
+
+        links_by_type, links_by_name = util.get_links(param_config.config_directory +
+                                                      param_config.country_config["links_file"])
+        for link in links_by_name.values():
+            logging.info(link)
+            form_1 = link["to_form"]
+            column_3 = link["to_condition"].split(":")[0]
```

Comment:
This logging statement could be more descriptive. Only `link` appears later a little messy in logs. 
`form_1` and `column_3` are not saying much about those variables. Could you change numbers to some words?